### ISSUE TITLE
Artist page vs Album Artist

### ISSUE THREAD

**Issue by ravtakhar** [jmshrv/finamp#994]
Currently the artists tab is pulling in the album artists rather than the actual artists tab like Jellyfin.

The artists page should contain all their albums, albums/singles they are part of from other artists, their top tracks, related artists and their most played tracks (by user).

![image](https://github.com/user-attachments/assets/238c85b5-ee61-490a-aea6-aec734158571)

**Comment by Chaphasilor** [comment]
Yes, that's a known issue. In offline more it really is the artists tab, so it's even inconsistent.  
Support for "Appears On" is also planned (#572)

**Comment by Chaphasilor** [comment]
@ravtakhar except for the "More Like This" section we've now implemented the necessary changes as part of #1145 :)
